code/modules/language/beachbum.dm

```diff
@@ -0,0 +1,21 @@
+/datum/language/beachbum
+	name = "Beach Bumbian"
+	desc = "For when you're too toasted to speak normally."
+	speech_verb = "mumbles"
+	ask_verb = "grills"
+	exclaim_verb = "hollers"
+	key = "u"
+	space_chance = 80
+	default_priority = 1000 //Haha!
```

Comment:
Would this mean that the curators will speak this ancient and forbidden tongue as their default language?